mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
Do we have a tooltip somewhere listing these values?
If not we probably should add them somewhere, if we do then I would think the extra complexity would be worth it since the war factory is in a special place (being so expensive)